src/components/20-molecules/link/index.story.js

```diff
@@ -0,0 +1,25 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+import './index';
+
+storiesOf('Link', module)
```

Comment:
Add the prefix which generates a folder in the "storybook" -> `storiesOf('/Atoms/Links')`